modules/library/coverage/src/main/java/org/geotools/image/ImageWorker.java

```diff
@@ -2051,7 +2686,25 @@ private final ImageWorker maskComponentColorModelByte(final Color transparentCol
      * @see InvertDescriptor
      */
     public final ImageWorker invert() {
-        image = InvertDescriptor.create(image, getRenderingHints());
+        // ParameterBlock creation
+        ParameterBlock pb = new ParameterBlock();
+        pb.setSource(image, 0);
+        if (JAIExt.isJAIExtOperation("algebric")) {
```

Comment:
The comment about using a constant is valid here too (similar one made in coverage operations)